yuuna/lib/src/pages/implementations/subtitle_options_dialog_page.dart

```diff
@@ -410,8 +515,11 @@ class _SubtitleOptionsDialogPage
       subtitleOptions.regexFilter = newRegexFilter;
       subtitleOptions.fontName = newFontName;
       subtitleOptions.fontSize = newFontSize;
+      subtitleOptions.fontColor = fontColor.value;
```

Comment:
Does the font and outline color change if I manually edit the text, and not use the picker? Can you verify if hex values entered are valid if I manually edit the text?